usage/workflow_download.py

```
@@ -40,6 +44,59 @@ def get_id_set(env_name, given):
+
+
+class WorkflowImageCompressPlugin(JmOptionPlugin):
+    plugin_key = 'workflow_image_compress'
+
+    def invoke(self, image, quality, **kwargs):
+        result = compress_image_in_place(image.save_path, quality)
+        if result is None:
+            return
+
+        original_size, compressed_size, replaced = result
+        action = '压缩完成' if replaced else '压缩后未变小，保留原图'
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)